snapcraft/internal/pluginhandler/_scriptlets.py

```diff
@@ -44,6 +44,14 @@ def _run(self, scriptlet):
             with suppress(FileNotFoundError):
                 os.unlink(scriptlet_path)
 
+    def prepare(self, *, scriptlet):
```

Comment:
Or hand it a few different scriptlets in `__init__` and made the `install()` and `prepare()` run them.